content/blog/2023/04/26/research-community-open-call.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Research Community Open Call #1 – April 2023"
+created: 2023-04-26
+categories:
+  - commmunity-call
+authors:
+  - lifexitself
+---
+
+This is the first in a series of monthly open meetings hosted by the Life Itself research community. These purpose of these calls is to provide an opportunity for newcomers to learn about the community and engage in discussions, and for community members to share updates about ongoing work and coordinate across interest groups.
+
+In this opening call, Rufus proposes an initial framework for a research agenda aligned with Life Itself’s wider purpose of paradigmatic transition. This approach is informed by the [[primacy-of-being|primacy of being]] thesis and is primarily focused on the topics of inner development and cultural evolution, with a secondary focus on systems transformation and the politics of paradigmatic change.
+
+![[excalidraw/research-topical-map-2023-01-14.excalidraw.svg]]
+
+> You can find more detailed information about the research community—including our approach, focus topics and ongoing projects—on our [[research|overview page]].
+
+### How to participate
+These calls are open for anyone to join. If you’re interested in participating or learning more:
+- When: 3pm CET on the second Friday of every month
+- Where: Google Meet [https://meet.google.com/rks-pfqv-dbz](https://meet.google.com/rks-pfqv-dbz)
+
+### Video
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
In theory you should just be able to post a youtube video link and it "just works" ... and this is fine too.

Reply:
FYI this feature requires this specific YouTube URL format or it will either not detect it or incorrectly parse it:

https://www.youtube.com/watch?v=VKKUBK4J6Ho